docs/examples/units-extension/relations/FrequencyRelations.java

```diff
@@ -1,4 +1,4 @@
-package qual;
+package relations;
```

Comment:
Why did this change package?

Reply:
Hmm upon further testing it was unnecessary to change the package. I've reverted this change.